recipes/responsible_ai/llama_guard/README.md

```diff
@@ -2,14 +2,14 @@
 <!-- markdown-link-check-disable -->
 Llama Guard is a language model that provides input and output guardrails for LLM deployments. For more details, please visit the main [repository](https://github.com/facebookresearch/PurpleLlama/tree/main/Llama-Guard).
 
-This folder contains an example file to run Llama Guard inference directly. 
+This folder contains example notebooks on running Llama Guard stand alone and validating Llama Guard performance against a reference dataset. The dataset is not provided, only the format in which it should be to use the scripts out of the box. Additionally, Llama Guard is being used as an optional safety checker when running the regular Llama [inference script](../../inference/local_inference/inference.py).
```

Comment:
Should we callout this can be used to convert ToxicChat dataset using script to run validation on LlamaGuard?

Reply:
This is not intended to convert the toxic chat dataset yet, but it's the base for that in the future.